util/crypto/crypto.go

```diff
@@ -0,0 +1,30 @@
+package crypto
+
+import "github.com/pactus-project/pactus/crypto/bls"
+
+func SignMessageWithPrivateKey(prv, msg string) (string, error) {
```

Comment:
Please don't define extra util when we have `crypto` package. Use crypto package.
Here we have a wrapped function.